crates/solidity/testing/snapshots/bindings_output/modifiers/simple/generated/0.5.0-success.txt

```diff
@@ -0,0 +1,37 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+References and definitions: 
+    ╭─[input.sol:1:1]
+    │
+  1 │ contract FunctionModifier {
+    │          ────────┬───────  
+    │                  ╰───────── def: 1
+  2 │     bool public locked;
+    │                 ───┬──  
+    │                    ╰──── def: 2
+    │ 
+  4 │     modifier noReentrancy() {
+    │              ──────┬─────  
+    │                    ╰─────── def: 3
+  5 │         require(!locked, "No reentrancy");
```

Comment:
Unrelated to this PR:

now that we have built-ins, what do you think of using `success` and `failure` file suffixes to indicate the status of binding (instead of parsing), as that is what we are really checking here.

i.e. If a file has syntax errors, but we still bind everything else, it is a success. But for a test like this, it will change from `0.4.11-failure` to `0.5.0-success` because of the unresolved `require`..
 
 We can still include the parse errors for debuggability though.

Thoughts?

Reply:
It shouldn't be the case that parsing fails and all bindings are resolved. At least I think that should never happen. If that could happen, I'd suspect that the test is not well written (as in, why does it contain syntax that's not important for testing the binding). I like the idea of tagging tests `success`ful only when bindings are resolved. What do you think about using the `success` suffix when both parsing and bindings succeed?

Reply:
parsing can still fail when we are explicitly testing binding in the precense of syntax errors, or when we are testing a syntax that is not available in all versions.
Binding here should run regardless and produce results.  Right?

Reply:
Yes, I'm referring to the meaning of `success`/`failure` mark only. There would be no  change in how the tests are executed.